evasquez26/angular4-startup
# S4 one 53 54 filter by all and open courses

## Story / Task

[ONE-53](https://bixlabs.atlassian.net/browse/ONE-53) and [ONE-53](https://bixlabs.atlassian.net/browse/ONE-54)

## What this PR does?

Allows the user to filter courses by all that are open and all courses. Also it allows to check courses by catagories.

### Before 

- We could not filter courses by category
- We could not filter courses if they were open for enrollment or by all courses

### After

- We can filter courses by category
- We can filter courses if they are open for enrollment or by all courses

### How to test / reproduce

- Log in into the app
- Click the courses option on the left menu
- Observe that there are images with the corresponding category
- Observe that there's a filter on the top right which allows you to pick from All courses, Open and Closed
- Clicking one of the categories should take you to a list of courses by that category
- Clicking details from that course

### TODOs

- [x] Rebase after merging #30 
- [ ] Apply continuous scrolling on Sprint 6 [ONE-79](https://bixlabs.atlassian.net/browse/ONE-79)

### Related PRs

N/A

## Migrations

NO
